.github/workflows/ci.yaml

```diff
@@ -8,10 +8,11 @@ jobs:
     runs-on: ubuntu-latest
     if: "!contains(github.event.head_commit.message, 'skip-ci')"
     container:
-      image: docker://bitcli/bit:latest
+      image: docker://bitcli/bit:0.0.364
     steps:
       - name: Bit config
         run: |
+          bit -v
```

Comment:
whats all this?

Reply:
David had an issue with symlink in bit, and it's happen to me also in the CI, so he wanted to know exactly with which version it's happened to me.
And because I'm using the latest docker image, we don't know.
So now I print the version of Bit in the beginning and I also changed the CI to work with version 364 at the moment because it's the last version that don't have this problem.